### PR TITLE
📦 Setup project

### PR DESCRIPTION
**Pull request by b-tarczynski**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1201905791425111

### CODE REVIEW

File: packages/contracts/.solhint.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "extends": "solhint:recommended",
+  "rules": {
+    "avoid-suicide": "error",
+    "compiler-version": [
+      "error",
+      ">=0.6.10"
```
Comment (msieczko) [inline packages/contracts/.solhint.json:7]:
```suggestion
      ">=0.8.10"
```

File: packages/contracts/.compiler.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "version": "0.8.10",
+  "settings": {
+    "optimizer": {
+      "enabled": true,
+      "runs": 20000
+    }
+  }
+}
```
Comment (msieczko) [inline packages/contracts/.compiler.json:9]: Delete if unused

File: packages/contracts/test/setup.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+export function setupFixtureLoader () {
+  if (!loadFixture) {
+    loadFixture = setupOnce()
+  }
+  return loadFixture
+}
+
+type CurrentLoader = { loader: FixtureLoader, returns: FixtureReturns, fixture: Fixture<any> }
+
+function setupOnce () {
```
Comment (msieczko) [inline packages/contracts/test/setup.ts:23]:
```suggestion
function setupOnce() {
```
Update our linter config 😅

**Review by msieczko — CHANGES_REQUESTED** [review]

File: packages/contracts/.compiler.json (hunk 2 above)
Comment (b-tarczynski) [reply packages/contracts/.compiler.json:9]: It's used in [hardhat.config.ts](https://github.com/TrueFiEng/devcon-raffle/blob/setup-project/packages/contracts/hardhat.config.ts)

**Review by msieczko — APPROVED** [review]

**Pull request merged**